hummingbot/connector/exchange/vertex/vertex_constants.py

```diff
@@ -0,0 +1,307 @@
+from typing import Any, Dict
+
+# A single source of truth for constant variables related to the exchange
+from hummingbot.core.api_throttler.data_types import LinkedLimitWeightPair, RateLimit
+from hummingbot.core.data_type.in_flight_order import OrderState
+
+# The max size of a digest is 66 characters (Vertex uses digests comprable to client order id).
+MAX_ORDER_ID_LEN = 66
+
+HEARTBEAT_TIME_INTERVAL = 30.0
+
+ORDER_BOOK_DEPTH = 100
+
+VERSION = "0.0.1"
+
+EXCHANGE_NAME = "vertex"
+
+DEFAULT_DOMAIN = "vertex"
+TESTNET_DOMAIN = "vertex_testnet"
+
+QUOTE = "USDC"
+
+BASE_URLS = {
+    DEFAULT_DOMAIN: "https://prod.vertexprotocol-backend.com",
+    TESTNET_DOMAIN: "https://test.vertexprotocol-backend.com",
+}
+
+WSS_URLS = {
+    DEFAULT_DOMAIN: "wss://prod.vertexprotocol-backend.com",
+    TESTNET_DOMAIN: "wss://test.vertexprotocol-backend.com",
+}
+
+CONTRACTS = {
+    DEFAULT_DOMAIN: "0xbbee07b3e8121227afcfe1e2b82772246226128e",
+    TESTNET_DOMAIN: "0x5956d6f55011678b2cab217cd21626f7668ba6c5",
+}
+
+CHAIN_IDS = {
+    DEFAULT_DOMAIN: 42161,
+    TESTNET_DOMAIN: 421613,
+}
+
+HBOT_BROKER_ID = ""
+
+SIDE_BUY = "BUY"
+SIDE_SELL = "SELL"
+
+TIME_IN_FORCE_GTC = "GTC"  # Good till cancelled
+TIME_IN_FORCE_IOC = "IOC"  # Immediate or cancel
+TIME_IN_FORCE_FOK = "FOK"  # Fill or kill
+TIME_IN_FORCE_POSTONLY = "POSTONLY"  # PostOnly
+
+# API PATHS
+POST_PATH_URL = "/execute"
+QUERY_PATH_URL = "/query"
+INDEXER_PATH_URL = "/indexer"
+SYMBOLS_PATH_URL = "/symbols"
+WS_PATH_URL = "/ws"
+WS_SUBSCRIBE_PATH_URL = "/subscribe"
+
+# POST METHODS
+PLACE_ORDER_METHOD = "place_order"
+PLACE_ORDER_METHOD_NO_LEVERAGE = "place_order_no_leverage"
+CANCEL_ORDERS_METHOD = "cancel_orders"
+CANCEL_ALL_METHOD = "cancel_product_orders"
+
+# REST QUERY API TYPES
+STATUS_REQUEST_TYPE = "status"
+ORDER_REQUEST_TYPE = "order"
+SUBACCOUNT_INFO_REQUEST_TYPE = "subaccount_info"
+MARKET_LIQUIDITY_REQUEST_TYPE = "market_liquidity"
+ALL_PRODUCTS_REQUEST_TYPE = "all_products"
+MARKET_PRICE_REQUEST_TYPE = "market_price"
+FEE_RATES_REQUEST_TYPE = "fee_rates"
+CONTRACTS_REQUEST_TYPE = "contracts"
+SUBACCOUNT_ORDERS_REQUEST_TYPE = "subaccount_orders"
+MAX_WITHDRAWABLE_REQUEST_TYPE = "max_withdrawable"
+
+# WS API ENDPOINTS
+WS_SUBSCRIBE_METHOD = "subscribe"
+TOB_TOPIC_EVENT_TYPE = "best_bid_offer"
+POSITION_CHANGE_EVENT_TYPE = "position_change"
+SNAPSHOT_EVENT_TYPE = "market_liquidity"
+TRADE_EVENT_TYPE = "trade"
+DIFF_EVENT_TYPE = "book_depth"
+FILL_EVENT_TYPE = "fill"
+POSITION_CHANGE_EVENT_TYPE = "position_change"
+
+# Products
+# NOTE: Index 7+ is only on testnet
+PRODUCTS = {
```

Comment:
this is not used too, is this intended to be a map between product id to symbol?

Reply:
This is no longer used, but provides a mapping for navigating if so inclined, happy to remove it though.

Reply:
@R-K-H no worries it's not going to affect the performance